### PR TITLE
Identity columns should have autoincrement property returned by getTables

### LINKED ISSUE #77: GetTables must return autoincrement property for identity columns
The UI needs to know which columns are Identity columns so that it can show them as incremental fetching candidates.

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #77

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/MSSQL.php [hunk 1]
```diff
@@ -439,7 +443,8 @@ private function quickTablesSql(): string
                   sys.columns.precision AS NUMERIC_PRECISION,
                   sys.columns.scale AS NUMERIC_SCALE,
                   sys.columns.max_length AS CHARACTER_MAXIMUM_LENGTH,
-                  pks.index_name AS pk_name
+                  pks.index_name AS pk_name,
+                  pks.is_identity AS is_identity
```
Comment (tomasfejfar) [inline src/Keboola/DbExtractor/Extractor/MSSQL.php]: isn't the same missing from \Keboola\DbExtractor\Extractor\MSSQL::fullTablesSql ?

**Comment by tomasfejfar** [comment]
I'm not totally confident I understand the whole picture, though. Especially the difference between quick and full tables fetch and why they do not need to return the same structure...

**Comment by pivnicek** [comment]
Well, they got split for performance reasons, and I'm not yet sure if they should be the same or not since the consumers are different.  The consumer for the quick getTables is the UI for the component, and the consumer for the full result is the storage metadata.  
The UI for the component does need this property, but the storage metadata probably does not.
Regardless, maybe we'll get rid of this difference and just use the simple one in the future, depending on the outcome of https://github.com/keboola/db-extractor-common/issues/98

**Comment by tomasfejfar** [comment]
The main problem is that this whole behavior is based on whether the $table parameter is empty and it then returns two (potentially wastly) different things. 

Anyway this PR LGTM.

**Pull request merged**